app-modules/form/src/Listeners/ClearAuthorFormSubmissionCountCache.php

```diff
@@ -45,7 +45,7 @@ class ClearAuthorFormSubmissionCountCache implements ShouldQueue
     public function handle(FormSubmissionCreated $event): void
     {
         if (! is_null($event->submission->author)) {
-            Cache::tags('form-submission-count')
+            Cache::tags('{{form-submission-count}}')
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.